codecparsers/bitReader.h

```diff
@@ -50,7 +50,7 @@ class BitReader {
     /* You are allowed to skip less than BitReader::CACHEBYTES bytes
```

Comment:
Could we sent other patch to change the behaviors, Why we can’t skip more than CACHEBYTES? This will force user know details about CACHEBYTES. 
Use a loop in skip, maybe better.